src/Microsoft.Dnx.Runtime.Abstractions/Constants.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
```

Comment:
These usings are not needed. 
Also, waiting for @troydai to point out that there is no license header.